### DISCUSSION TITLE
如何给静态网站做客户端搜索 | J实验室

### DISCUSSION THREAD

**Discussion by giscus** [weijunext/new-blog#3] [General]
# 如何给静态网站做客户端搜索 | J实验室

全栈工程师，Next.js开源手艺人，掘金签约作者，折腾才有未来

https://weijunext.com/article/adding-client-side-search-to-static-site

**Comment by wwfalcon** [comment]
meilisearch 也不错 ;)

前端可以用这个 @meilisearch/instant-meilisearch

https://www.npmjs.com/package/@meilisearch/instant-meilisearch

**Comment by flashlab** [comment]
这些都太重了，个人博客就算写了100w字也才2Mb数据，如果想尽量搜的全就直接拉取文本大包到本地搜索，或者直接跳转到谷歌必应。另外咱就是说，做好分类和标签的博客，搜索功能只是锦上添花。
